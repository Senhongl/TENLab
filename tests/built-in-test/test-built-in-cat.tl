a = [[1,2,3],[5,6,7]];
b = [[10,11,12],[14,15,16]];
c = cat(a,b,0);
print(c);
c = cat(a,b,1);
print(c);