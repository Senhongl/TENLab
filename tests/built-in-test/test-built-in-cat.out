  1   2   3
  5   6   7
 10  11  12
 14  15  16
[ CPUIntType{4,3} ]
  1   2   3  10  11  12
  5   6   7  14  15  16
[ CPUIntType{2,6} ]
